docs/running-on-mesos.md

```diff
@@ -520,7 +520,7 @@ See the [configuration page](configuration.html) for information on Spark config
   <td><code>spark.mesos.checkpoint</code></td>
   <td>false</td>
   <td>
-    If set, agents running tasks started by this framework will write the framework pid, executor pids and status updates to disk. 
+    If set to true, the agents that are running the spark-executors will write framework pids (Spark), executor pids and status updates to disk. 
```

Comment:
nits:

s/spark/Spark

remove the '-'

remove "(Spark)".  All of this data applies to Spark, not just the framework pid.

s/pids/pid (there's only one framework)